validator/docs/schemas.md

```diff
@@ -0,0 +1,163 @@
+  - name: cronjob-name  # Process name - Required
+    is_cronjob: true    # Defining a scheduled process (cronjob) - Required
+    start_command: npm run worker # Executed command when the pod is started - Required
+```
+
+### Scales
+When you deploy an application in a Kubernetes environment, you define how many replicas you would like to run. 
+Each replica represents a Kubernetes POD that encapsulates your application container. 
+And also you can increase, or reduce the number of replicas in the given limit.
+
+```yaml
+scales:
+  my-process:         # Process to scale
+    min-replicas: 1   # Minimal number of pods, INTEGER - Required
+    max-replicas: 10  # Minimal number of pos, INTEGER - Required
+    targetCPUUtilizationPercentage: # Percentage of CPU which will trigger a horizontal scaling when it is exceeded, INTEGER, 0 <= x <= 100 - Required
+  another-process: ...
+```
+
+### Crons
+Crons are tasks that run at a specific time or interval. You can use this type of features to automate tasks, commonly as: reporting, backups, maintenance, emails and so on.
+
+You can define the crons information with the following structure.
+
+```yaml
+crons:
+  my-cronjob: # Cronjob to define, it must be the same name as the cronjob defined in processes
+    concurrency_policy: 'Forbid' # Allow, Forbid or Replace concurrency of cronjobs, STRING - Required. [Kubernetes documentation](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy)
+    schedule: '* * * * *' # STRING - Required
+    suspend:              # BOOLEAN
+  another-resource: ...
+```
+
+### Resources
+This structure defines the resources to be allocated for your application.
+
+```yaml
+resources:
+  my-resource:  # Name of the resource to define
+    limit:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    limits:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    request:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+    requests:
+      cpu:      # STRING / NUMBER
+      memory:   # STRING
+  another-resource: ...
+```
+
+
+### TemplateVars
+Define variables used in configuration templates when building the kubernetes configuration before deployment. Here are the available variables:
+
+_Note_: All the variables should be prefixed by `tpl`.
+
+```yaml
+templateVars:         # Variables used in conf templates
+  tplCriticity:       # Define if the app is on the critical path, STRING - high, low, none
+  tplPublicServices:  # Define if the application web is public accessible or not, BOOLEAN
+  tplExpandedTimeout: # Define the timeout on the ingress controller, STRING
+  tplSessionAffinity: # Activate or deactivate session affinity on ingress controller, BOOLEAN
+  tplTerminationGracePeriod: # Define POD termination grace period, INTEGER
+  tplCanary: # Define name for canary, STRING
+```
+
+### Dependencies
+The list of dependencies used by your project.
```

Comment:
Even as a Kapten developer, I am not sure what this list is used for. We should either remove it, if there is no use, or dig a bit more to find out why we need this and document it more.